packages/framework/src/client.ts

```diff
@@ -291,7 +292,21 @@ export class Client {
         setResult(stepResult);
       }
 
-      return stepResult.outputs as any;
+      if (
```

Comment:
Instead of checking on specific channels, should we check if `stepResult.outputs` is typeof `string`?

Reply:
I think I like the check to be a bit more verbose so any future of us will easier understand what the check is for :)